### PR TITLE
Bug 1813185: Add stable-4.4, fast-4.4 and candidate-4.4 channel for 4.4 release

### PR DESCRIPTION
**Pull request by jhadvig**
/assign @spadgett

### CODE REVIEW

File: frontend/public/module/k8s/cluster-settings.ts [hunk 1]
```diff
@@ -31,6 +31,9 @@ export const getAvailableClusterChannels = () => ({
   'stable-4.3': 'stable-4.3',
   'fast-4.3': 'fast-4.3',
   'candidate-4.3': 'candidate-4.3',
```
Comment (spadgett) [inline frontend/public/module/k8s/cluster-settings.ts]: We should remove the 4.3 channels in 4.4.

**Comment by spadgett** [comment]
@jhadvig I'd suggest updating the channels to 4.5 in master, then we can clone the bug back to 4.4.

**Comment by spadgett** [comment]
Sorry, disregard my comment. I didn't realize this was the release-4.3 branch.

**Comment by jwforres** [comment]
@sdodson @LalatenduMohanty bringing in OTA folks to coordinate - this PR is what would make the 4.4 channels appear in someone's 4.3 admin console

**Comment by sdodson** [comment]
Those channels were populated yesterday
https://github.com/openshift/cincinnati-graph-data/pull/111
https://github.com/openshift/cincinnati-graph-data/pull/114
Additionally, empty channels for 4.5 were populated as well so we could go ahead and update master too, though it's going to be a a long time before anything shows up in those channels.
https://github.com/openshift/cincinnati-graph-data/pull/113

**Comment by jhadvig** [comment]
@spadgett @jwforres  can we go ahead and merge this one ?

**Pull request merged**